lib/application/index.js

```diff
@@ -4,6 +4,9 @@
  * configuration and rendering.
  */
 
+// Local
+var Promise = require('../polyfills/promise');
```

Comment:
1kb is insignificant enough that I don't feel strongly about it. I defer to your judgement on this one 😺.